packages/teleport-component-generator-react/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@teleporthq/teleport-component-generator-react",
+  "version": "0.7.0",
```

Comment:
@alexnm i ideal way will be we keep all our generators as 0.6 and then we use ``lerna publish`` so it will bump to 0.7 and push to npm and tag the releases. In this way if we do a publish it will bump to 0.8 or a path or prerelease or something and then it pushes them. We can do a test run with this with verdaccio and confirm

Reply:
feel free to update the version directly on this branch. I will check it out later as I'm not at my laptop now